// [[Reason: Invalid computed property can only be detected by TSC]] ////

//// [tests/cases/conformance/parser/ecmascript6/ComputedPropertyNames/parserComputedPropertyName17.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -4,18 +4,14 @@
 declare var v: invalid;
 
 /// [Errors] ////
 
-parserComputedPropertyName17.ts(1,15): error TS9014: Computed properties must be number or string literals, variables or dotted expressions with --isolatedDeclarations.
 parserComputedPropertyName17.ts(1,16): error TS2304: Cannot find name 'e'.
 parserComputedPropertyName17.ts(1,19): error TS9009: At least one accessor must have an explicit return type annotation with --isolatedDeclarations.
 
 
-==== parserComputedPropertyName17.ts (3 errors) ====
+==== parserComputedPropertyName17.ts (2 errors) ====
     var v = { set [e](v) { } }
-                  ~~~
-!!! error TS9014: Computed properties must be number or string literals, variables or dotted expressions with --isolatedDeclarations.
-!!! related TS9027 parserComputedPropertyName17.ts:1:5: Add a type annotation to the variable v.
                    ~
 !!! error TS2304: Cannot find name 'e'.
                       ~
 !!! error TS9009: At least one accessor must have an explicit return type annotation with --isolatedDeclarations.
